discretisedfield/mesh.py

```diff
@@ -380,11 +430,11 @@ def __iter__(self):
             yield self.index2point(index)
 
     def axis_points(self, axis, /):
-        warnings.warn("Deprecated; use `mesh.midpoints` instead.", FutureWarning)
-        return getattr(self.midpoints, axis)
+        warnings.warn("Deprecated; use `mesh.points` instead.", FutureWarning)
+        return getattr(self.points, axis)
 
     @functools.cached_property
-    def midpoints(self):
+    def points(self):
```

Comment:
Maybe add an additional `midpoints` and raise an `AttributeError` pointing to `points`.